src/edu/ucsb/nceas/metacat/restservice/v2/MNResourceHandler.java

```diff
@@ -1319,7 +1319,7 @@ private void getLog() throws InvalidToken, ServiceFailure, NotAuthorized, Invali
         }
 
         logMetacat.debug("calling getLogRecords");
-        Log log = MNodeService.getInstance(request)
+        org.dataone.service.types.v2.Log log = MNodeService.getInstance(request)
```

Comment:
I added new imported  class `org.apache.commons.logging.Log` since each ResourceHandler class has its own log object. So there are duplicate Log class and I have to use the full name here.